odo/backends/sql.py

```diff
@@ -103,7 +103,7 @@
     mssql.UNIQUEIDENTIFIER: string,
     # The SQL Server TIMESTAMP value doesn't correspond to the ISO Standard
     # It is instead just a binary(8) value with no relation to dates or times
-    mssql.TIMESTAMP: bytes_,
+    # mssql.TIMESTAMP: bytes_,
```

Comment:
So if you just run that little bit of code #567, you'll get the behavior. 

```python 

In [1]: import sqlalchemy as sa
   ...: from odo import discover
   ...: columns = [sa.Column('amount', sa.REAL),
   ...:            sa.Column('ds', sa.TIMESTAMP)]
   ...: dta = sa.Table('t', sa.MetaData(), *columns)
   ...: discover(dta)
   ...:
Out[1]: dshape("var * {amount: ?float32, ds: ?bytes}")
```
Note that I never specified to sqlalchelmy that this was running on MSSQL. The problem is that sqlalchemy's `mssql.TIMESTAMP` datatype is matching all timestamps... not just MSSQL's. 

If you want to maintain this behavior, then we'll be telling the entire blaze ecosystem whenever you encounter something that looks like a `mssql.TIMESTAMP`--even if its not on MSSQL--to become `bytes`.

There may be a way to limit `revtypes` so that the MSSQL special ones only occur when you specify that you are running on MSSQL. I'm not sure how to do that (or if its feasible).